uw-frame-components/js/app-config.js

```diff
@@ -33,15 +33,11 @@ define(['angular'], function(angular) {
         .value('SEARCH', {
             'searchURL': '/web/apps/search/',
         })
-        .value('FEATURES', {
-            'enabled': false,
-            'groupFiltering': false,
-            'serviceURL': 'staticFeeds/features.json',
-        })
-        .value('NOTIFICATION', {
-            'enabled': true,
+        .value('MESSAGES', {
+            'announcementsEnabled': true,
+            'notificationsEnabled': true,
             'groupFiltering': true,
```

Comment:
Maybe we don't need a `groupFiltering` toggle -- whether it filters on groups could be implied by whether there's groups data in the messages feed. Tell it what groups to filter to and it filters; leave the groups out and it doesn't.